static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -45,9 +82,10 @@ function UserActionsNavigator({event, crumbs}: Props) {
         {userActionCrumbs.map(item => (
           <PanelItemCenter
             key={item.id}
-            className={
-              currentUserAction && currentUserAction.id === item.id ? 'selected' : ''
-            }
+            isHovered={closestUserAction?.id === item.id}
+            isSelected={currentUserAction?.id === item.id}
+            onMouseEnter={() => onMouseEnter(item)}
+            onMouseLeave={() => onMouseLeave()}
```

Comment:
do these anonymous functions, and also the `onMouseEnter` / `onMouseLeave` functions get recreated on each render?
If a prop like `currentTimestamp` changes does every item in the list re-render?